core/contracts/financial_templates/FeePayer.sol

```diff
@@ -70,6 +70,22 @@ contract FeePayer is Testable {
         return regularFee.add(latePenalty);
     }
 
+    /**
+     * @notice Pays UMA DVM final fees to the Store contract. This is a flat fee charged for each price request.
+     * @return the amount of collateral that was paid to the Store.
+     */
+    function payFinalFees() public returns (FixedPoint.Unsigned memory totalPaid) {
+        StoreInterface store = StoreInterface(finder.getImplementationAddress("Store"));
```

Comment:
nit: you can merge master and use the _getStore() method:
```
StoreInterface store = _getStore();
```

Reply:
nice